packages/core/src/lib/TwaManifest.ts

```diff
@@ -297,7 +297,7 @@ export class TwaManifest {
       features: {
         locationDelegation: {enabled: DEFAULT_ENABLE_LOCATION},
       },
-      shareTargetJson: JSON.stringify(webManifest['share_target']),
+      shareTarget: undefined, // webManifest['share_target'],
```

Comment:
I don't really like checking in commented out code. Potentially what we could have is a comment linking to a bug or issue explaining why it's undefined.

Reply:
Removed the commented-out code and added a comment. I want to avoid having broken things in `master`.

Reply:
LGTM